src/client/window.guardian.d.ts

```diff
@@ -15,6 +17,15 @@ declare global {
     };
     Cypress: unknown;
     ga: unknown;
+    grecaptcha: {
```

Comment:
Add typings for the global `grecaptcha` object populated by the google Recaptcha script